src/assets/styles/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$mainRed: #940003;
+$mainGray: #424242;
+
+$breakpoint-small: 530px;
```

Comment:
use the same case for variables. `$breakpoint-medium` is kebab-case, while `$mainGray` is camelCase. For Sass files the most common is kebab-case.